packages/utils/src/numbers.ts

```diff
@@ -0,0 +1,17 @@
+import BN from 'bignumber.js';
+export { BN };
+
+export function amountToInt(amount: string, decimals: number): string {
+  return new BN(amount).times(10 ** decimals).dp(0).toFixed();
+}
+
+export function amountToDecimal(amount: string, decimals: number): string {
+  return new BN(amount).div(10 ** decimals).toFixed();
+}
+
+export function truncateNumber(n: string, sd: number = SIGNIFICANT_DIGITS, roundingMode?: BN.RoundingMode) {
+  return new BN(n).sd(sd, roundingMode).toFixed();
+}
+
+export const SIGNIFICANT_DIGITS = 7;
```

Comment:
Why 7?

Reply:
This file is a copy/paste from another codebase I use very often. It's not very required here yet, but could be useful later. 
I usually use truncateNumber to display amount on frontends, cutting too long text values. 
Can remove this also if you feel it's not necessary to have it here.